src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/translations/messages.en.yml

```diff
@@ -199,15 +199,34 @@ mail.confirmation.publish_production.service_name: Service name
 mail.confirmation.publish_production.publish_date: Publish date
 mail.confirmation.publish_production.team_id: Team id
 mail.confirmation.publish_production.subject: 'Production connection has been requested (%entityId%)'
-service.overview.progress.label.intake-conducted: Intake done
-service.overview.progress.label.entity-on-test: Entity on test
-service.overview.progress.label.contract-signed: Contract signed
-service.overview.progress.label.representative-approved: Approved by representative
-service.overview.progress.label.privacy-questions: Privacy questions answered
-service.overview.progress.label.production-connection: Production connection aquired
-service.overview.progress.tooltip.intake-conducted: Is the intake done
-service.overview.progress.tooltip.entity-on-test: Is there an entity on the test environment
-service.overview.progress.tooltip.contract-signed: Is a contract signed
-service.overview.progress.tooltip.representative-approved: Is the service approved by a representative
-service.overview.progress.tooltip.privacy-questions: Are the privacy questions answered
-service.overview.progress.tooltip.production-connection: Is a production connection active
+service.overview.entitylist.title: Entities of service %name%
+service.overview.entitylist.test.title: Entities @ test environment
+service.overview.entitylist.production.title: Entities @ production environment
+service.overview.entitylist.name: Name
+service.overview.entitylist.entityId: Entity ID
+service.overview.entitylist.protocol: Protocol
+service.overview.entitylist.state: State
+service.overview.entitylist.empty: No entities found.
+service.overview.legend.success: Done
+service.overview.legend.in-progress: In progress
+service.overview.legend.info: Unstarted
+service.overview.progress.label.intake-conducted: "Intake\ndone"
+service.overview.progress.label.entity-on-test: "Entity\non\ntest"
+service.overview.progress.label.contract-signed: "Contract\nsigned"
+service.overview.progress.label.representative-approved: "Approved\nby\nrepresentative"
+service.overview.progress.label.privacy-questions: "Privacy\nquestions\nanswered"
+service.overview.progress.label.production-connection: "Production\nconnection\naquired"
+service.overview.progress.tooltip.intake-conducted.success: "Thank you\nthe intake was completed"
+service.overview.progress.tooltip.intake-conducted.info: "Is the intake done?"
+service.overview.progress.tooltip.entity-on-test.success: 'Thank you<br> there is a entity on the test environment<br><a href="/">more info</a>'
+service.overview.progress.tooltip.entity-on-test.info: "Is there an entity on the test environment?"
```

Comment:
In order to render wysiwyg editors in the translation interface, please append a .html section to the key. Otherwise html tags will be stripped.

Reply:
To test this: first import the translations

`bin/console lex:trans:import`

Then search for the appropriate translation in the web translator and update it.

Finally invalidat the cahces with the magic blue button.

Reply:
FYI: the html tags will not be stripped you only don't get the fancy editor. Addee `.html` to the key to show the WYSIWYG-editor.